src/ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_process_net_command_json.py

```diff
@@ -115,21 +116,39 @@ def process_net_command_json(self, py_db, json_contents):
 
         DEBUG = False
 
-        request = self.from_json(json_contents, update_ids_from_dap=True)
+        try:
+            request = self.from_json(json_contents, update_ids_from_dap=True)
+        except KeyError as e:
+            request = self.from_json(json_contents, update_ids_from_dap=False)
+            error_msg = str(e)
```

Comment:
could we use `traceback.format_exception_only` to get the message here?

Reply:
Not really... 

```
>>> exc = KeyError('Wrong ID sent from the client: %s' % (dap_id,))
>>> print(str(exc))
'Wrong ID sent from the client: 11'  # <-- I want to take out those single quotes
>>> print(traceback.format_exception_only(exc.__class__, exc)[-1])
KeyError: 'Wrong ID sent from the client: 11'  # <-- Single quotes still there with exception and additional new line.

>>>
```